zubhub_frontend/zubhub/src/App.js

```diff
@@ -112,10 +64,26 @@ const LazyImport = props => {
   );
 };
 
+const ProtectedRoute = ({ component: LazyComponent, ...props }) => {
```

Comment:
This checks if the user is authenticated by access the `token` and redirects to the login page if not.